### ISSUE TITLE
New source: use Navi cheatsheats

### ISSUE THREAD

**Issue by Seirdy** [chubin/cheat.sh#181]
[denisidoro/navi](https://github.com/denisidoro/navi) is a trending cheatsheet viewer/manager that uses its own cheatsheets along with custom user-made ones. It uses [this cheatsheet repo](https://github.com/denisidoro/cheats) by default. Other repositories exist, some of which are listed in its [`featured_repos.txt`](https://github.com/denisidoro/cheats/blob/master/featured_repos.txt) file.

The file format is described in Navi's [README](https://github.com/denisidoro/navi/blob/master/README.md#cheatsheet-syntax).

**Comment by Seirdy** [comment]
The file format seems pretty simple; there's a chance I could implement this and send a PR over spring break or later. Having remote classes helps 😉.

**Comment by chubin** [comment]
@Seirdy Yes, I know about this project; it seems to be pretty interesting; one of my colleagues @bobkodex actively use it.

You could integrate cheat sheets of navi at cheat.sh first, and then we could try to integrate it into navi (in the opposite direction)
